zheng-common/src/test/java/com/zheng/common/util/PaginatorTest.java

```diff
@@ -0,0 +1,62 @@
+package com.zheng.common.util;
+
+import static org.mockito.Matchers.anyDouble;
+import com.diffblue.deeptestutils.Reflector;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import java.lang.reflect.InvocationTargetException;
+
+@RunWith(PowerMockRunner.class)
+@PowerMockIgnore({"javax.management.*"})
+public class PaginatorTest {
+
+	/* testedClasses: Paginator */
+	@Test
+	public void initTotalPageOutputVoid() {
+		final Paginator paginator = new Paginator();
+		paginator.initTotalPage();
+		Assert.assertEquals(0L, paginator.getTotalPage());
+	}
+
+	@PrepareForTest({Paginator.class, Math.class})
+	@Test
+	public void getHtmlOutputNotNull() throws Exception,
+			InvocationTargetException {
+		PowerMockito.mockStatic(Math.class);
+		final Paginator paginator = new Paginator();
+		Reflector.setField(paginator, "total", 0L);
```

Comment:
We don't want to see reflection in our tests. We might be able to work around this using constructor/getters/setters.